frontend/src/components/pages/schemas/Schema.Details.tsx

```diff
@@ -166,121 +171,124 @@ class SchemaDetailsView extends PageComponent<{ subjectName: string }> {
-                                ]}
-                                rowKey="name"
-                                dataSource={fields}
-                                pagination={false}
-                                style={{
-                                    maxWidth: '100%',
-                                    marginTop: '1.5rem',
-                                    marginBottom: '1.5rem',
-                                }}
+                                        disabled={versions.length == 0}
+                                >
+                                    {versions.map(v => <Select.Option value={v} key={v}>Version {v} {v == versions[versions.length - 1] ? '(latest)' : null}</Select.Option>)}
+                                </Select>
+                            </Label>
+
+                            <Label text="Details" style={{alignSelf: 'stretch'}}>
+                                <div style={{display: 'inline-flex', flexWrap: 'wrap', minHeight: '32px', alignItems: 'center', rowGap: '.3em'}}>
+                                    {Object.entries({
+                                        'Type': type,
+                                        'Name': name,
+                                        'Namespace': namespace,
+                                    }).map(([k, v]) => {
+                                        if (!k || v === undefined || v === null) return null;
+                                        return <Tag key={k}><span style={{color: '#2d5b86'}}>{k}:</span> {toSafeString(v)}</Tag>
+                                    })}
+                                    {!!doc && <a href={doc}>
+                                        <Tag style={{cursor: 'pointer'}}><span style={{color: '#2d5b86'}}>Documentation:</span> <a style={{textDecoration: 'underline'}} href={doc}>{doc}</a></Tag>
+                                    </a>}
+                                </div>
+
+                            </Label>
+                        </div>
+
+                        <div style={{marginBottom: '1.5em', display: 'flex', gap: '1em'}}>
+                            <OptionGroup label=""
+                                         options={{
+                                             'Show Fields': 'fields',
+                                             'Show JSON': 'json',
+                                         }}
+                                         value={uiSettings.schemaDetails.viewMode}
+                                         onChange={s => uiSettings.schemaDetails.viewMode = s}
                             />
-                        }
 
-                    </div>
-                </Section>
-            </PageContent>
+                            <NoClipboardPopover placement="top">
+                                <div>
+                                    {/* the additional div is necessary because popovers do not trigger on disabled elements, even on hover */}
```

Comment:
[question] Is this still applicable?

Reply:
You mean checking the availability of a clipboard? The comment about `isDisabled` is still applicable.
While I think it's not absolutely necessary to check for an availability of this API, I didn't want to remove functionality as part of this PR. btw it's supported in almost all browsers according to caniuse.